components/widgets/admin_console/admin_panel_with_link.tsx

```diff
@@ -19,11 +19,13 @@ type Props = {
     disabled: boolean;
     linkTextId: string;
     linkTextDefault: string;
+    linkDataTestId?: string;
```

Comment:
I mentioned this above but I would like if this was `linkId` since I dont know if we want to expose `testId` as a component prop